src/arch/armv8/aarch64/boot.S

```diff
@@ -76,8 +76,11 @@ _reset_handler:
 1:
 	cmp x5, x3
 	b.ge	2f
-	ldrb w6, [x4]
-	add x4, x4, #1
+#if PLAT_CLUSTERS_CORES_NUM_SIZE != 8
+#error "struct platform cluster core_num field type size is not 8"
+#endif
```

Comment:
I would prefer to define the value 8 as a macro that represents the platform word size.

Reply:
@josecm thoughts?

Reply:
Maybe I'm being to pedantic, but my issue was to give more clarity to the meaning of the value. But it's fine by me, the warning message is probably enough.